demos/serverless/src/handlers.js

```diff
@@ -23,8 +24,8 @@ const chime = new AWS.Chime({ region: 'us-east-1' });
 chime.endpoint = new AWS.Endpoint(endpoint);
 
 const chimeSDKMeetings = new AWS.ChimeSDKMeetings({ region: currentRegion });
-if(endpoint != 'https://service.chime.aws.amazon.com'){
-  chimeSDKMeetings.endpoint = new AWS.Endpoint(endpoint);
+if(chimeSDKMeetingsEndpoint != 'https://service.chime.aws.amazon.com' && useChimeSDKMeetings === 'true'){
```

Comment:
if `useChimeSDKMeetings` is false but `chimeSDKMeetingsEndpoint` is set then what does that mean? Can we just always respect `chimeSDKMeetingsEndpoint` and if it is set then automatically set `useChimeSDKMeetings` to true?

Reply:
In that case, we will be using Global Meetings. Also, we can not do that because we are defaulting value of chimeSDKMeetingsEndpoint to https://service.chime.aws.amazon.com, and we will not have any way to distinguish between when to use Global Meetings Client and Chime SDK meetings client for Prod.